src/main/java/org/synyx/urlaubsverwaltung/publicholiday/PublicHoliday.java

```diff
@@ -0,0 +1,54 @@
+package org.synyx.urlaubsverwaltung.publicholiday;
+
+import de.jollyday.Holiday;
+import org.synyx.urlaubsverwaltung.period.DayLength;
+
+import java.util.Objects;
+
+/**
+ * Describes a public holiday with the UV specific setting if it is a half public holiday or not.
+ * (e.g. christmas eve can be configured to be a NOON public holiday for instance.)
+ */
+public final class PublicHoliday {
+
+    private final Holiday holiday;
```

Comment:
könnte man jollyday hier noch weiter verstecken, sodass man es nicht in die App leaked?